posts/2023-state-of-rust-analysis.md

```diff
@@ -0,0 +1,74 @@
+![2](https://github.com/rust-lang/blog.rust-lang.org/assets/45044840/dd27ddff-fd85-43ac-81ef-09a4e65d30ae)
+
+## Rust Usage at Work
+The growing maturation of Rust can be observed through the increased number of organizations utilizing the language in 2022. In fact, an impressive 29.7% stated that they use Rust for the majority of their coding work at their workplace. This marks a significant 51.8% increase compared to the previous year.
+![3](https://github.com/rust-lang/blog.rust-lang.org/assets/45044840/bae6d0ea-1bc2-4edc-a119-c665ce9abc5e)
+
+There are numerous reasons why we are seeing increased use of Rust in professional environments; 86% of respondents opted to use Rust at work because it enables their teams to build relatively correct and bug-free software, 84% were particularly impressed with its performance advantages, and 69% were most compelled by Rust’s considerable security and safety advantages. While respondents reported many practical benefits to using Rust on professional engineering teams, we were also pleased to find that 76% of respondents continue to use Rust simply because they found it fun and enjoyable.
+
+![4](https://github.com/rust-lang/blog.rust-lang.org/assets/45044840/cb17b676-5904-4dbd-8cc9-31ee7aa94c7f)
+
+Of those respondents that used Rust at work, 72% reported that it helped their team achieve its goals (a 4% increase from the previous year) and 75% have plans to continue using it on their teams in the future. 
+
+But like any language being applied in the workplace, Rust’s learning curve is an important consideration; 39% of respondents using Rust in a professional capacity reported the process as “challenging” and 9% of respondents said that adopting Rust at work has “slowed down their team”. However, 60% of productive users felt Rust was worth the cost of adoption overall. 
+![5](https://github.com/rust-lang/blog.rust-lang.org/assets/45044840/c533954b-eadc-41c0-ac9e-717d7ebc6e22)
+
+It is exciting to see the continued growth of professional Rust usage and the confidence so many users feel in its performance, control, security and safety, enjoyability, and more!
+
+## Supporting the Future of Rust
+
+A key goal of the State of Rust survey is to shed light on challenges, concerns, and priorities Rustaceans are currently sitting with. 
+
+Of those respondents who shared their main worries for the future of Rust, 26% have concerns that the developers and maintainers behind Rust are not properly supported – a decrease of more than 30% from the previous year’s findings. While we would of course like to see this number at 0%, this improvement reflects the growing maturity of the Rust Foundation and the support they provided the Project in 2022 during its first full year in operation.
```

Comment:
I'm not sure if this is the right channel to note this, but this statement:

> 26% have concerns that the developers and maintainers behind Rust are not properly supported – a decrease of more than 30% from the previous year’s findings.

Does not necessarily lead to this conclusion:

> this improvement reflects the growing maturity of the Rust Foundation and the support they provided the Project in 2022 during its first full year in operation.

While certainly the Foundation's involvement in supporting Rust maintainers and project members is important and shouldn't be understated, I was surprised by the wording the first time I read through this. There are other reasons why this number may have dropped -- for example, this is gauging sentiment from the Rust community at large, and not (if I'm understanding the data correctly) reporting the feelings of Rust project members and Rust maintainers themselves (who are a small percentage of this population as a whole). Also, project members and maintainers do receive other channels of support, such as partial or full compensation to work on Rust-related activities from employers and non-employer sponsorship such as Github sponsors and patreon.

Given all of that, I think this should be weakened to not make such a direct claim. Perhaps it's better to just use wording like "suggests", or back it up with other data that leads to this specific conclusion.